src/enforcement/enforcer.ts

```diff
@@ -90,44 +107,130 @@ export class Enforcer implements IEnforcer {
    * @throws {@link PermitConnectionError} if an error occurs while sending the authorization request to the PDP.
    * @throws {@link PermitPDPStatusError} if received a response with unexpected status code from the PDP.
    */
-  public async check(
-    user: IUser | string,
-    action: IAction,
-    resource: IResource | string,
+  public async bulkCheck(
+    checks: Array<[IUser | string, IAction, IResource | string]>,
```

Comment:
it would have been better to use an object here instead of array.
it's nicer developer experience with parameter name completion.

Reply:
will fix before merging

Reply:
changed